libs/net/dal/Services/SourceService.cs

```diff
@@ -145,6 +153,13 @@ public Source UpdateAndSave(Source entity, bool updateChildren = false)
     {
         var original = FindById(entity.Id) ?? throw new NoContentException("Entity does not exist");
         this.Context.UpdateContext(original, entity, updateChildren);
+        original.MediaTypeSearchMappingsManyToMany.Clear();
```

Comment:
This is too noisy.  We have example of how to only delete the values that should be removed.